### PR TITLE
[CALCITE-5166] Method accept(RelShuttle) is not overridden in LogicalCalc and LogicalTableModify

### PR DESCRIPTION
**Pull request by rkondakov**
[https://issues.apache.org/jira/browse/CALCITE-5166](https://issues.apache.org/jira/browse/CALCITE-5166)

### CODE REVIEW

File: core/src/main/java/org/apache/calcite/rel/RelHomogeneousShuttle.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
 import org.apache.calcite.rel.core.TableFunctionScan;
 import org.apache.calcite.rel.core.TableScan;
 import org.apache.calcite.rel.logical.LogicalAggregate;
+import org.apache.calcite.rel.logical.LogicalCalc;
 import org.apache.calcite.rel.logical.LogicalCorrelate;
```
Comment (chunweilei) [inline core/src/main/java/org/apache/calcite/rel/RelHomogeneousShuttle.java:23]: Tips: remove the `.` in the commit message.

File: core/src/main/java/org/apache/calcite/rel/logical/LogicalTableModify.java [hunk 2]
```diff
@@ -92,4 +93,8 @@ public static LogicalTableModify create(RelOptTable table,
         sole(inputs), getOperation(), getUpdateColumnList(),
         getSourceExpressionList(), isFlattened());
   }
+
+  @Override public RelNode accept(RelShuttle shuttle) {
+    return shuttle.visit(this);
+  }
```
Comment (chunweilei) [inline core/src/main/java/org/apache/calcite/rel/logical/LogicalTableModify.java:99]: Could you please check whether other operators have the same issue too?

File: core/src/main/java/org/apache/calcite/rel/RelHomogeneousShuttle.java (hunk 1 above)
Comment (rkondakov) [reply core/src/main/java/org/apache/calcite/rel/RelHomogeneousShuttle.java:23]: Done. `.` is not in commit message anymore.

File: core/src/main/java/org/apache/calcite/rel/logical/LogicalTableModify.java (hunk 2 above)
Comment (rkondakov) [reply core/src/main/java/org/apache/calcite/rel/logical/LogicalTableModify.java:99]: Other operators don't have the same issue. These two operators had overloaded methods in `RelShuttle` but didn't have overridden `accept()` methods. Other logical operators neither had overloaded methods in `RelShuttle` nor `accept()` methods (or have both options).

Comment (chunweilei) [reply core/src/main/java/org/apache/calcite/rel/logical/LogicalTableModify.java:99]: Get it! Thank you for your effort.

**Review by chunweilei — APPROVED** [review]
LGTM

**Pull request merged**